Tetris/src/tetris/GameSystem.java

```diff
@@ -0,0 +1,102 @@
+package tetris;
+
+import java.util.ArrayList;
+
+/**
+ * Class for holding and managing individual game components
+ * 
+ * This class handles the main data & logic of the game, including
+ * collision detection, movement & rotation, deletion of rows, score calculation,
+ * tetromino creation and management
+ * 
+ * These all interact to form the "state" of the game
+ * 
+ */
+public class GameSystem {
+	private int score;
+	private ArrayList<Tetromino> nextShapeQueue;
+	private Tetromino activeTetromino;
+	
+	public boolean blockGrid[][];
+	
+	/**
+	 * Default constructor for game system
+	 * 
+	 * @return game system object
+	 * 
+	 */
+	public GameSystem(){
+		// Set score to zero
+		
+		// Game board is 22 blocks high by 10 blocks wide
+		
+		// Initialize Tetrominos
```

Comment:
@tatiana5kova 
we would have to create some initial tetrominos with random shapes to fill our nextShapeQueue
Tetromino myTetromino = new Tetromino('l');
etc